data/discord_messages.yaml

```diff
@@ -986,34 +847,7 @@ vef:
       Example: auto, manual, yaml, legacy (not lovelace)
 
     Examples: 0.84.3/venv/yaml
-check:
```

Comment:
This is one of the most used bot commands, we should keep this.

But maybe remove

```
    ***Always*** run the configuration check *command* when you make changes.
    Don't trust the UI check - it misses some problems.
```